apis/swagger.yml

```diff
@@ -584,6 +584,25 @@ paths:
         500:
           $ref: "#/responses/500ErrorResponse"   
 
+  /task/metrics:

```

Comment:
I am not sure about this name, is there a better url for this?

Reply:
I think that's OK. FYI https://apiguide.readthedocs.io/en/latest/build_and_publish/use_RESTful_urls.html#